src/module/loader/module-resolver.cpp

```diff
@@ -153,7 +160,11 @@ ModuleInfo ModuleResolver::ModuleResolverImpl::Resolve(const char* name, const c
     // Normalize module context path.
     filesystem::Path basePath =
         (path == nullptr) ? filesystem::CurrentDirectory() : filesystem::Path(path);
-    
+
+    if (fromContent) {
+        return ResolveFromContent(name, basePath);
```

Comment:
I don't have a strong preference on extracting path resolution for content into caller, or make it in resolver.

Since now it's just basePath  / name concatenation, I can extract it into caller.